projects/payment-lib/src/lib/components/add-remission/add-remission.component.spec.ts

```diff
@@ -30,10 +33,24 @@ describe('AddRemissionComponent', () => {
     amount: new FormControl(),
     refundReason: new FormControl(),
     refundDDReason: new FormControl(),
-    reason: new FormControl()
+    reason: new FormControl(),
+    feesList: new FormControl([
+      { refund_amount: 100, selected: 1 },
+      { refund_amount: 50, selected: 0 }
+    ])
   });
 
-  form.setValue({remissionCode:"HWF-A1B-23C",amount:"100",refundReason:"Test Refund  Reason",refundDDReason:"Default Reason",reason:"Test Reason"});
+  form.setValue({
+    remissionCode:"HWF-A1B-23C",
+    amount:"100",
+    refundReason:"Test Refund  Reason",
+    refundDDReason:"Default Reason",
+    reason:"Test Reason",
+    feesList: [
+      { refund_amount: 100, selected: 1 },
```

Comment:
Is this redundant if it's defined below?

Reply:
without this, tests are failing